pkg/network/vmispec/interface_test.go

```diff
@@ -101,6 +101,38 @@ var _ = Describe("VMI network spec", func() {
 		Entry("more then one interface", vmiSpecInterfaces(iface1, iface2, iface3), []string{iface1, iface2, iface3}),
 	)
 
+	Context("pop interface by network", func() {
+		const netName = "net1"
+		network := podNetwork(netName)
+		expectedStatusIface := v1.VirtualMachineInstanceNetworkInterface{Name: netName}
+		expectedStatusIfaces := vmiStatusInterfaces(iface1, iface2)
```

Comment:
I placed it outside the table since those definitions should be created only once, no reason to recreate it for each entry in the table. I will replace it right before the table definition.

Reply:
I do not think we have an efficiency requirement in unit tests. IMO we should have things clearer, that is a higher priority.
When the variables are placed far away, one may think it is used in other places (at least I did look to see where this is happening).

In this case the cost is negligible, it is not that we perform I/O or CPU intense operation.

Reply:
fixed